Add Heltec Lora V3  with Sim800l modem.

Oganisyan opened the pull request.

gereic commented:
I removed the SIM800l because it has only 2G connection and this will be removed in the next years.
So the question is, how long it will work.
What do you want to do ?

nepat commented:
Would recommend to use 7080G module for low power consumption, support of low supply voltage and IOT protocols LTE-M and NB-IOT.

Oganisyan commented:
Is Ok
I change to A7600 Module.



Am Mo., 7. Apr. 2025 um 09:54 Uhr schrieb gereic ***@***.***>:

> I removed the SIM800l because it has only 2G connection and this will be
> removed in the next years.
> So the question is, how long it will work.
> What do you want to do with this board?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/gereic/GXAirCom/pull/192#issuecomment-2782348740>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACEIAVEHIYNYROOXRK56JK32YIVMZAVCNFSM6AAAAAB2BIPPACVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDOOBSGM2DQNZUGA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
> [image: gereic]*gereic* left a comment (gereic/GXAirCom#192)
> <https://github.com/gereic/GXAirCom/pull/192#issuecomment-2782348740>
>
> I removed the SIM800l because it has only 2G connection and this will be
> removed in the next years.
> So the question is, how long it will work.
> What do you want to do with this board?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/gereic/GXAirCom/pull/192#issuecomment-2782348740>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACEIAVEHIYNYROOXRK56JK32YIVMZAVCNFSM6AAAAAB2BIPPACVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDOOBSGM2DQNZUGA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
